### PR TITLE
initial prototype of ORTModule integration

### PR DESCRIPTION
**Pull request by souptc**
Initial prototype to use eager mode with ortmodule together.

### CODE REVIEW

**Review by thiagocrepaldi — APPROVED** [review]
Assu

File: src/eager/ort_eager.cpp [hunk 1]
```diff
@@ -2,12 +2,45 @@
...
+}
+}
 
 namespace torch_ort {
 namespace eager {
+using namespace onnxruntime::training;
+
+py::object ORTTensor_toDLPack(const at::Tensor& data)
+{
+  //const at::Tensor& ort_at_tensor = THPVariable_Unpack(data);
```
Comment (chandru-r) [inline src/eager/ort_eager.cpp:30]: Nit: delete commented line

File: src/eager/ort_eager.cpp [hunk 2]
```diff
@@ -2,12 +2,45 @@
...
+  OrtValue ort_value = create_ort_value(data);
+  return py::reinterpret_steal<py::object>(onnxruntime::training::framework::torch::ToDlpack(ort_value));
```
Comment (chandru-r) [inline src/eager/ort_eager.cpp:32]: Given that you have a using namespace onnxruntime::training consider shortening to framework::torch::ToDlpack

File: src/eager/ort_eager.cpp [hunk 3]
```diff
@@ -2,12 +2,45 @@
...
+}
+
+at::Tensor ORTTensor_FromDLPack(py::object dlpack_tensor)
```
Comment (chandru-r) [inline src/eager/ort_eager.cpp:35]: Can this be a  const reference?

File: src/eager/python/ortmodule/torch_inline_extensions/.ninja_log [hunk 4]
```diff
@@ -0,0 +1,61 @@
+# ninja log v5
```
Comment (chandru-r) [inline src/eager/python/ortmodule/torch_inline_extensions/.ninja_log:1]: Do we need to check-in the ninja files? Did you accidentally include them?

File: src/eager/python/ortmodule/torch_inline_extensions/build.ninja [hunk 5]
```diff
@@ -0,0 +1,29 @@
+ninja_required_version = 1.3
```
Comment (chandru-r) [inline src/eager/python/ortmodule/torch_inline_extensions/build.ninja:1]: Did you mean to checkin this?

File: src/eager/python/ortmodule/torch_inline_extensions/.ninja_log (hunk 4 above)
Comment (souptc) [reply src/eager/python/ortmodule/torch_inline_extensions/.ninja_log]: no it is accidently got included...

**Review by chandru-r — APPROVED** [review]

**Pull request merged**